docs/handbook/systems-administration.md

```diff
@@ -481,14 +481,15 @@ To configure your system to use system repository, perform the following actions
 # svcadm enable svc:/application/pkg/zones-proxyd:default
 ```
 
-2) In NGZ:
+2) In non-global zone:
 
 ```
+# pkg install pkg:/package/pkg/zones-proxy
```

Comment:
@Mno-hime is it correct? After merging and reading it I wonder if it should be zones-proxy-client

Reply:
It is correct, sorry for the noise.